proto/src/determined/rbac/v1/rbac.proto

```diff
@@ -218,6 +218,8 @@ message UserRoleAssignment {
   int32 user_id = 1;
   // the role and scope of the assignment
   RoleAssignment role_assignment = 2;
+  // optional list of users to apply the role.
+  repeated int32 user_ids = 3;
```

Comment:
Do we need this change?

Can we instead just submit multiple ```UserRoleAssignment``` (one request still since ```AssignRolesRequest``` takes an array)

Reply:
was floating the idea, but no it isn't necessary to do multiple user assignments in one requests